<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>uiTestDataCreatWidget</class>
 <widget class="QWidget" name="uiTestDataCreatWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>77</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>uiTestDataCreatWidget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,1,1,1,1">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_GPSID">
       <property name="text">
        <string>GPSID</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_CenterCoordinate">
       <property name="text">
        <string>Lat,Lng</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_DataAmount">
       <property name="text">
        <string>DataAmount</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_creatData">
       <property name="text">
        <string>creatDataFile</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
